src/statd/statd.c

```diff
@@ -129,7 +129,7 @@ static json_t *json_get_ip_link(void)
 }
 
 static int ly_add_yanger_data(const struct ly_ctx *ctx, struct lyd_node **parent,
-			      char *model, const char *arg)
+                              char *model, const char *arg)
```

Comment:
These still look incorrect to me?

Checkpatch.pl also complains:
```
git show 211921 | ./output/build/linux-6.5.11/scripts/checkpatch.pl

ERROR: code indent should use tabs where possible                                                                                                                                        
#34: FILE: src/statd/statd.c:132:                                                                                                                                                        
+                              char *model, const char *arg)$
```

Reply:
i

> These still look incorrect to me?
> 
> Checkpatch.pl also complains:
> 
> ```
> git show 211921 | ./output/build/linux-6.5.11/scripts/checkpatch.pl
> 
> ERROR: code indent should use tabs where possible                                                                                                                                        
> #34: FILE: src/statd/statd.c:132:                                                                                                                                                        
> +                              char *model, const char *arg)$
> ```
That is a result of emacs linux-c-mode i think
Tab-with is set to 8 and there is 6 spaces, 

> These still look incorrect to me?
> 
> Checkpatch.pl also complains:
> 
> ```
> git show 211921 | ./output/build/linux-6.5.11/scripts/checkpatch.pl
> 
> ERROR: code indent should use tabs where possible                                                                                                                                        
> #34: FILE: src/statd/statd.c:132:                                                                                                                                                        
> +                              char *model, const char *arg)$
> ```
I used emacs with linux-c and our clang-formatter on the code... Will check the formatting script in infix.